index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.1/dist/css/bootstrap.min.css"
+      rel="stylesheet"
+      integrity="sha384-+0n0xVW2eSR5OomGNYDnhzAbDsOXxcvSN1TPprVMTNDbiYZCxYbOOl7+AMvyTG2x"
+      crossorigin="anonymous"
+    />
+    <link rel="stylesheet" href="css/main.css" />
+    <title>Library</title>
+</head>
+<body>
+    <h2 class="topic">Library</h2>
+    <button id="addBook" type="submit" class="btn btn-primary">New Book</button>
+    
+
+    <form id="forms"  style="visibility: hidden;">
+        <label for="name1">Name:</label><br>
+        <input type="text" id="name1" name="name1" placeholder="Enter Book Name"><br>
+
+        <label for="author">Author:</label><br>
+        <input type="text" id="author" name="author" placeholder="Enter Author Name"><br>
+
+        <label for="pages">No. Of Pages:</label><br>
+        <input type="text" id="pages" name="pages" placeholder="No. Of Pages"><br>
+
+        <input type="radio" id="read1" name="read1">
+        <label for="readen">Read?</label><br>
+
+        <button type="submit">Submit</button>
+    </form> 
```

Comment:
### [OPTIONAL]

- [ ] It is always good to add validations to your form. This ensures that the user can not create an empty book. Also for the number of pages, the `TYPE` should be a number, not text.